fastfetch 2.0.0 tries to fetch yyjson at build time.

This is not allowed in Gentoo.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,24 +36,6 @@
 
 include(CheckIncludeFile)
 
-include(FetchContent)
-function(ff_fetch_dep package repo tag)
-    FetchContent_Declare(
-        "${package}"
-        GIT_REPOSITORY "${repo}"
-        GIT_TAG "${tag}"
-        GIT_PROGRESS TRUE
-    )
-    FetchContent_GetProperties("${package}")
-    if(NOT ${package}_POPULATED)
-        message("-- Fetching dependency ${package}@${tag} from ${repo}")
-        FetchContent_Populate(${package})
-        add_subdirectory(${${package}_SOURCE_DIR} ${${package}_BINARY_DIR} EXCLUDE_FROM_ALL)
-    endif()
-endfunction()
-
-ff_fetch_dep(yyjson "https://github.com/ibireme/yyjson" "0.7.0")
-
 #####################
 # Configure options #
 #####################
